docs/course_materials/exercises/01_Linux/check_class_exercise_1.py

```diff
@@ -0,0 +1,51 @@
+import argparse
+import os.path
+from os import path
+from pathlib import Path
+
+home = str(Path.home())
+exercise_commands = ['mkdir test', 'mkdir tmp', 'tree']
+
```

Comment:
what if mkdir will be run with some additional arguments,
mdkir -p test/tmp  

also, what if the command runs correctly but wasn't created? or created in the wrong place? 
I think that in this case, it's enough to check if the directories exist

Reply:
I wanted to check for mkdir to make sure the terminal was used. 
If you think it's not needed I will remove it.